src/controllers/krate/versions.rs

```diff
@@ -365,6 +397,12 @@ struct ResponseMeta {
     release_tracks: Option<ReleaseTracks>,
 }
 
+#[derive(Serialize)]
+struct ReleaseTracksMeta {
```

Comment:
```suggestion
struct ListVersionsMeta {
```

in case we ever want to add something else to it too 😉

Reply:
Marked as resolved due to adopting the "always include meta" approach.